wasmedge-piper/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "wasmedge-piper"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde_json = "1.0.118"
+wasmedge-wasi-nn = "0.7.1"
```

Comment:
We could update `wasmedge-wasi-nn` to `0.8.0`.

Reply:
ok, https://github.com/second-state/WasmEdge-WASINN-examples/pull/145/commits/85cebe0efdd749c821fba248a1e75baf567021a7